systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaUtils.java

```diff
@@ -61,18 +61,29 @@
 
     private KafkaUtils() {}
 
-    public static void waitForKafkaReady(String clusterName) {
-        waitForKafkaStatus(clusterName, Ready);
+    public static boolean waitForKafkaReady(String namespaceName, String clusterName) {
+        return waitForKafkaStatus(namespaceName, clusterName, Ready);
     }
 
-    public static void waitForKafkaNotReady(String clusterName) {
-        waitForKafkaStatus(clusterName, NotReady);
+    public static boolean waitForKafkaReady(String clusterName) {
+        return waitForKafkaStatus(kubeClient().getNamespace(), clusterName, Ready);
     }
 
-    public static void waitForKafkaStatus(String clusterName, Enum<?>  state) {
-        String namespace = kubeClient().getNamespace();
-        Kafka kafka = KafkaResource.kafkaClient().inNamespace(namespace).withName(clusterName).get();
-        ResourceManager.waitForResourceStatus(KafkaResource.kafkaClient(), kafka, state);
+    public static boolean waitForKafkaNotReady(String namespaceName, String clusterName) {
+        return waitForKafkaStatus(namespaceName, clusterName, NotReady);
+    }
+
+    public static boolean waitForKafkaNotReady(String clusterName) {
+        return waitForKafkaStatus(kubeClient().getNamespace(), clusterName, NotReady);
+    }
```

Comment:
Shouldn't we keep just methods which has `namespace` as needed parameter? Or we'll keep it for tests which will not run in parallel namespaces?

Reply:
That's correct.